src/Output/Copy/Serializer/ContentType/XmlContentTypeSerializer.php

```diff
@@ -0,0 +1,373 @@
+<?php
+
+declare(strict_types=1);
+
+namespace DoclerLabs\ApiClientGenerator\Output\Copy\Serializer\ContentType;
+
+use DoclerLabs\ApiClientGenerator\Output\Copy\Schema\SerializableInterface;
+use DOMDocument;
+use DOMElement;
+use DOMNode;
+use Psr\Http\Message\StreamInterface;
+
+class XmlContentTypeSerializer implements ContentTypeSerializerInterface
+{
+    const MIME_TYPE = 'application/xml';
+    const ATTRIBUTE_NAMESPACE = 'xmlns';
+    const ATTRIBUTE_NAMESPACE_SEPARATOR = ':';
+
+    protected array       $config     = [
```

Comment:
it's /Copy/ file so needs to be php 7.0 compatible, so no type hints for properties

Reply:
👍🏼  yeah, need to be extra careful